.changeset/clever-swans-talk.md

```diff
@@ -0,0 +1,6 @@
+---
+"@shopware-ag/meteor-component-library": patch
+"nuxt-app": patch
+---
+
+Re-export mt-search and reuse it in nuxt-app example
```

Comment:
Please only include what's relevant for users of the component library. They just care about the `mt-search` component but not wether the component is used inside the nuxt example app.